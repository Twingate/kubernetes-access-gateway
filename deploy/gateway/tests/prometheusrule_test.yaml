suite: Prometheus Rule
templates:
  - prometheusrule.yaml
tests:
  - it: should enable Prometheus Rule
    set:
      metrics:
        prometheusRule:
          enabled: true
          rules:
            - alert: TestAlert
              expr: vector(1)
    asserts:
      - matchSnapshot: {}
