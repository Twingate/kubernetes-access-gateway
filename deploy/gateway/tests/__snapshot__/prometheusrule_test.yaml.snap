should enable Prometheus Rule:
  1: |
    apiVersion: monitoring.coreos.com/v1
    kind: PrometheusRule
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: kubernetes-access-gateway
        app.kubernetes.io/version: 0.1.0
        helm.sh/chart: kubernetes-access-gateway-0.1.0
      name: RELEASE-NAME-kubernetes-access-gateway
    spec:
      groups:
        - name: RELEASE-NAME-kubernetes-access-gateway
          rules:
            - alert: TestAlert
              expr: vector(1)
